CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Included the npub in the properties list sent to analytics.
 - Removed the like and repost counts from the Main and Profile feeds.
 - Replaced hard-coded color values.
+- Fixed typos.
```

Comment:
When we do a release, the Unreleased section of the CHANGELOG automatically becomes our Release Notes in TestFlight. Then we remove anything that's not user-facing so users aren't confused. So, if we update this line, it'll be easier to recognize this as something to remove from our public-facing release notes in TestFlight.

```suggestion
- Fixed typos in doc comments.
```

Reply:
Maybe we could annotate them more explicitly with a tag of some kind so that this kind of thing would get stripped automatically? Something like this:
`- Fixed typos in doc comments. [skip]`

Reply:
ooooh, I like that idea

Reply:
I've also seen CHANGELOGs with category headings like they show on https://keepachangelog.com/en/1.1.0/ -- we could go that route and have a section for `### Internal` or whatever for notes to exclude from release notes. But for now, I'd probably go with `[skip-release-notes]` on each line to be a little more clear about what skipping means here.